src/makeRobot.js

```diff
@@ -38,7 +38,56 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    version,
+    wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version},`
+        + ` wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(value = 1) {
+      if (value > 0) {
+        this.coords.y += value;
```

Comment:
```suggestion
    goForward(step = 1) {
      if (step > 0) {
        this.coords.y += step;
```

Имя переменной должно описывать что в ней хранится. В других методах аналогично